TU20Bot/Configuration/Controllers/AdministratorController.cs

```diff
@@ -0,0 +1,70 @@
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+using EmbedIO;
+using EmbedIO.Routing;
+using EmbedIO.WebApi;
+
+using MongoDB.Driver;
+
+using TU20Bot.Models;
+
+namespace TU20Bot.Configuration.Controllers {
+    public class AdministratorController : ServerController {
+        private readonly string[] administrator = { "Admin" };
+        
+        [Route(HttpVerbs.Post, "/create")]
+        public async Task create([QueryField] string username, [QueryField] string password) {
+            if (username == null || password == null)
+                throw HttpException.BadRequest();
+            
+            var validated = AuthorizationModule.validatePermissions(
+                administrator, Request.Headers["Authorization"], server.config.jwtSecret);
+            
+            if (!validated)
+                throw HttpException.Forbidden();
+
+            var collection = server.client.database
```

Comment:
Consider it done. Sounds like there's some unwanted code duplication.

Reply:
Sorry, I just fired up my IDE and went down to the delete method. They seemed to be named consistently?

Do you really think reverting the name is necessary? I like to keep with short one-word names when it doesn't impact code readability.

I changed the name of collection to something a bit shorter in this PR because the `user` essentially implied in the context and purpose of the class, generic types and surrounding variables.
<img width="599" alt="Screen Shot 2021-02-04 at 6 18 04 PM" src="https://user-images.githubusercontent.com/32211852/106967593-716ee800-6715-11eb-9f00-f4c8d429e03f.png">

Reply:
Yes, updating the name would help improve legibility, especially now that the variable is in a higher scope of the file.